src/sentry/api/endpoints/group_notes_details.py

```diff
@@ -26,6 +27,13 @@ def delete(self, request: Request, group, note_id) -> Response:
         except Activity.DoesNotExist:
             raise ResourceDoesNotExist
 
+        comment_deleted.send_robust(
```

Comment:
Is there any concern about sending a comment deleted webhook and then the comment not deleting for some reason?

Reply:
Well I wasn't until this moment :p I chose to send the webhook just before the note is deleted because otherwise I can't send the note object along - I guess I could pull off the relevant data (which is almost all of what's in it) before it's deleted and save that in a dictionary, delete the note, assert it's deleted, and then send the webhook, but I'd have to change the other ones to send a dictionary and not the model object, and change the functions receiving that data as well. Do db operations to delete fail often enough for this to be a concern?

Reply:
If there's a FK dependency on the comment or if there's 2 in-flight deletion requests. Either way, I don't think we have to solve this.

Reply:
@AniketDas-Tekky you made me paranoid so this last commit goes out to you